xbmc/TextureCache.h

```diff
@@ -80,8 +84,8 @@ class CTextureCache : public CJobQueue
    \sa CTextureCacheJob::CacheTexture
    */
   std::string CacheImage(const std::string& image,
-                         CTexture** texture = NULL,
-                         CTextureDetails* details = NULL);
+                         std::unique_ptr<CTexture>* texture = nullptr,
```

Comment:
` = nullptr` is not needed, std::unique_ptr will set its internal handle to null in its constructor